xbmc/storage/MediaManager.cpp

```diff
@@ -64,23 +64,22 @@ const char MEDIA_SOURCES_XML[] = { "special://profile/mediasources.xml" };
 CMediaManager::CMediaManager()
 {
   m_bhasoptical = false;
-  m_platformStorage = nullptr;
+  m_platformStorage.reset();
```

Comment:
does it need to be reset upon construction when there‘s the default member initializer?

Reply:
Nop shouldn't be needed, I had it removed before

Reply:
Adjusted